cachi2/core/package_managers/yarn/main.py

```diff
@@ -133,19 +134,6 @@ def _check_yarn_cache(source_dir: RootedPath) -> None:
     pass
 
 
-def _fetch_dependencies(source_dir: RootedPath, output_dir: RootedPath) -> None:
-    """Fetch dependencies using 'yarn install'.
-
-    :param source_dir: the directory in which the yarn command will be called.
-    :param output_dir: the directory where the yarn dependencies will be downloaded to.
-    :raises YarnCommandError: if the 'yarn install' command fails.
```

Comment:
We should probably move this "raises" docstring to the `_resolve_yarn_project` function.

Reply:
Since I kept `_fetch_dependencies` around what I'll do is that I'll also copy this raises docs note to `_resolve_yarn_project` as well.